README.md

```diff
@@ -148,6 +148,13 @@ The shared storage worklet invocation methods (`addModule`, `run`, and `selectUR
         *   `resolveToConfig` (defaults to false), a boolean denoting whether the returned promise resolves to a fenced frame config.
             *   If `resolveToConfig` is false or not specified, the returned promise resolves to an opaque URN that can be rendered by an iframe.
             *   If `resolveToConfig` is true, the returned promise resolves to a [fenced frame config](https://github.com/WICG/fenced-frame/blob/master/explainer/fenced_frame_config.md).
+        *   `savedQuery` (defaults to the empty string), a string naming the query to be saved or reused.
+            *   If the value of `savedQuery` is nonempty and has not previously been associated with a [result index](#result-index) for call to `selectURL()` on the same page, and if the call to `selectURL()` succeeds:
+                *    The pair of (`savedQuery`, [`index`](#result-index)) will be stored for the lifetime of the page.
+                *    The shared storage data-owning origin's site can reuse the query from anywhere within the page.
```

Comment:
let's use consistent terminology and call it the data origin.

Reply:
Done.